### ISSUE TITLE
C++ Runtime Assertion error

### ISSUE THREAD

**Issue by RobTheRealLifeAnimoo** [steamgriddb/steamgriddb-manager#63]
Hi. Got this weird error when hitting import or import all button. I already tried reinstalling C++ runtimes and looking the error up on google but it didn't help. I have attached a screenshot of the error.
![screenshot](https://user-images.githubusercontent.com/57270276/71183856-00524400-2279-11ea-8554-79f6e63c6512.png)

**Comment by a5066987** [comment]
Same problem

**Comment by kaankeskin** [comment]
Same here

**Comment by ricochet511** [comment]
Same issue here, tried a reboot/reinstall without success.

**Comment by RobTheRealLifeAnimoo** [comment]
> Same issue here, tried a reboot/reinstall without success.

Yeah. I tried clearing the %appdata% folders and registry. None of those things helped either.
